tests/perf/actor_timer/actor_timer_test.go

```diff
@@ -30,20 +30,36 @@ var tr *runner.TestRunner
 func TestMain(m *testing.M) {
 	testApps := []kube.AppDescription{
 		{
-			AppName:        "testapp",
-			DaprEnabled:    true,
-			ImageName:      "perf-actorjava",
-			Replicas:       4,
-			IngressEnabled: true,
-			AppPort:        3000,
+			AppName:           "testapp",
+			DaprEnabled:       true,
+			ImageName:         "perf-actorjava",
+			Replicas:          4,
+			IngressEnabled:    true,
+			AppPort:           3000,
+			DaprCPULimit:      "4.0",
+			DaprCPURequest:    "0.1",
+			DaprMemoryLimit:   "512Mi",
+			DaprMemoryRequest: "250Mi",
```

Comment:
We talked a little bit on a call about this: Setting this to 250Mi is specifically to prevent changes to the performance test environment. We probably want to set this to the default later.